h5p-bildetema/src/components/SearchPage/SearchFilter/SearchFilterDialog.tsx

```diff
@@ -81,9 +82,13 @@ const SearchFilterDialog = ({
               </button>
             </DialogTitle>
             <div className={styles.searchFilterWrapper}>
-              <ul className={styles.searchFilter}>
+              <ul role="list" className={styles.searchFilter}>
                 {topics?.map(topic => (
-                  <li key={topic.id} className={styles.checkBoxWrapper}>
+                  <li
+                    role="listitem"
```

Comment:
The role="listitem" for <LI> tag is til li-elementer, already has that as their default role making it kinda unesserary. However, for <ul> unsortedlist it help the semantic markup

Reply:
Yes, we add it because we use `display: flex` for instance in the css that often overrides the role that is set initially, like for `listitem` it has `display: list-item`